analyzers/slither/.deepsource/issues/SLITHER-W1001.toml

```diff
@@ -0,0 +1,30 @@
+title = "Storage abiencoderv2 array"
```

Comment:
```suggestion
title = "Storage abiencoderv2 array"
verbose_name = "0-0-abiencoderv2-array"
```

Reply:
also, for any keywords or anything code, use backticks.

"Storage `abiencoderv2` array"

Reply:
I am passing the title as-is from Slither. Some are in markdown, some are plaintext. Not modifying it from my side.

Reply:
Added the `verbose_name` in all issue titles.